index.html

```diff
@@ -1,6 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+    <link crossorigin="use-credentials" rel="manifest" href="/manifest.json">
```

Comment:
All of these URLs assume we're served from the server root, but the production deployment is in a subdirectory, https://soundscape-community.github.io/soundscape-web-client/. We can just make these start with a dot to make them relative paths.

Reply:
amazing !!! thank @steinbro .... I've just tested (very briefly) and it seems to work... I can save it in my phone and it looks like an app....